Marlin/src/lcd/language/language_uk.h

```diff
@@ -299,46 +300,48 @@ namespace Language_uk {
-  PROGMEM Language_Str MSG_MOVE_10MM                       = _UxGT("Рух по 10мм");
+  PROGMEM Language_Str MSG_HOTEND_TOO_COLD                 = _UxGT("Сопло надто холодне");
+  PROGMEM Language_Str MSG_MOVE_N_MM                       = _UxGT("Рухатися %sмм");
+  PROGMEM Language_Str MSG_MOVE_01MM                       = _UxGT("Рухатися 0.1мм");
+  PROGMEM Language_Str MSG_MOVE_1MM                        = _UxGT("Рухатися 1мм");
+  PROGMEM Language_Str MSG_MOVE_10MM                       = _UxGT("Рухатися 10мм");
   PROGMEM Language_Str MSG_SPEED                           = _UxGT("Швидкість");
   PROGMEM Language_Str MSG_BED_Z                           = _UxGT("Z Столу");
   PROGMEM Language_Str MSG_NOZZLE                          = _UxGT("Сопло, ") LCD_STR_DEGREE "C";
   PROGMEM Language_Str MSG_NOZZLE_N                        = _UxGT("Сопло ~");
   PROGMEM Language_Str MSG_NOZZLE_PARKED                   = _UxGT("Сопло запарковане");
-  PROGMEM Language_Str MSG_NOZZLE_STANDBY                  = _UxGT("Сопло очикує");
+  PROGMEM Language_Str MSG_NOZZLE_STANDBY                  = _UxGT("Сопло очікує");
   PROGMEM Language_Str MSG_BED                             = _UxGT("Стіл,  ") LCD_STR_DEGREE "C";
-  PROGMEM Language_Str MSG_CHAMBER                         = _UxGT("Камера,") LCD_STR_DEGREE "C";
-  PROGMEM Language_Str MSG_FAN_SPEED                       = _UxGT("Охолодження");
-  PROGMEM Language_Str MSG_FAN_SPEED_N                     = _UxGT("Охолодження ~");
+  PROGMEM Language_Str MSG_CHAMBER                         = _UxGT("Термокамера,") LCD_STR_DEGREE "C";
   #if LCD_WIDTH > 21
-    PROGMEM Language_Str MSG_STORED_FAN_N                  = _UxGT("Збережене охолодж. ~");
-    PROGMEM Language_Str MSG_EXTRA_FAN_SPEED               = _UxGT("Додаткове охолодж.");
-    PROGMEM Language_Str MSG_EXTRA_FAN_SPEED_N             = _UxGT("Додаткове охолодж. ~");
-    PROGMEM Language_Str MSG_CONTROLLER_FAN                = _UxGT("Контролер охолодження");
-  #else
-    PROGMEM Language_Str MSG_STORED_FAN_N                  = _UxGT("Збереж.охолодж. ~");
-    PROGMEM Language_Str MSG_EXTRA_FAN_SPEED               = _UxGT("Додат. охолодж.");
-    PROGMEM Language_Str MSG_EXTRA_FAN_SPEED_N             = _UxGT("Додат.охолодж ~");
-    PROGMEM Language_Str MSG_CONTROLLER_FAN                = _UxGT("Контролер охолодж.");
+    PROGMEM Language_Str MSG_FAN_SPEED                     = _UxGT("Швидкість вентилятору");
+    PROGMEM Language_Str MSG_FAN_SPEED_N                   = _UxGT("Швидкість вентилятору ~");
+    PROGMEM Language_Str MSG_STORED_FAN_N                  = _UxGT("Збережений вентилятор ~");
+    PROGMEM Language_Str MSG_EXTRA_FAN_SPEED               = _UxGT("Дод. швидкість вентилятору");
+    PROGMEM Language_Str MSG_EXTRA_FAN_SPEED_N             = _UxGT("Дод. швидкість вентилятору ~");
+    PROGMEM Language_Str MSG_CONTROLLER_FAN                = _UxGT("Вентилятор контроллеру");
```

Comment:
Here, "LCD_WIDTH > 21" usually only means 23 characters. These should probably just use the short versions.